testsuite/MDAnalysisTests/analysis/test_hydrogenbondautocorrel.py

```diff
@@ -23,121 +23,137 @@
 
 import pytest
 import six
-from six.moves import zip, range
+from six.moves import range
 
-from unittest import TestCase
-from MDAnalysisTests.datafiles import TRZ, TRZ_psf, PRM, TRJ
+from MDAnalysisTests.datafiles import TRZ, TRZ_psf
 from MDAnalysisTests import tempdir
-from numpy.testing import assert_array_almost_equal, assert_
+from numpy.testing import assert_almost_equal
 import numpy as np
 import mock
 
 import MDAnalysis as mda
 from MDAnalysis.analysis.hbonds import HydrogenBondAutoCorrel as HBAC
 
 
-class TestHydrogenBondAutocorrel(TestCase):
-    def setUp(self):
-        u = self.u = mda.Universe(TRZ_psf, TRZ)
-        self.H = u.atoms.select_atoms('name Hn')
-        self.N = u.atoms.select_atoms('name N')
-        self.O = u.atoms.select_atoms('name O')
-        self.excl_list = (np.array(range(len(self.H))),
-                          np.array(range(len(self.O))))
-
-    def tearDown(self):
-        del self.H
-        del self.N
-        del self.O
-        del self.u
-        del self.excl_list
+class TestHydrogenBondAutocorrel(object):
+    @pytest.fixture()
+    def u(self):
+        return mda.Universe(TRZ_psf, TRZ)
+
+    @pytest.fixture()
+    def H(self, u):
```

Comment:
Could you rename this fixture `hydrogens`, please? Let's avoid abusing single letter variable names.